l10n_ch_qr_bill/models/account_invoice.py

```diff
@@ -279,13 +278,3 @@ def _validate_qrr(self):
                 _("""The payment reference is not a valid QR Reference.""")
             )
         return True
-
-    def action_invoice_open(self):
-        res = super(AccountInvoice, self).signal_workflow('invoice_open')
-        for rec in self:
-            # Define reference once number has been generated
-            if rec.type == "out_invoice":
-                if not rec.name and rec.partner_bank_id._is_qr_iban():
-                    rec.name = rec.l10n_ch_qrr
```

Comment:
IIRC this was there to ensure the full workflow works for reconciliation. It most likely doesn't make sens for v8.

If you want a full reconciliation process you might need a `transaction_id` glue module for that with `base_transaction_id`. (like it was done with ISR)